coral/src/app/features/components/filters/useFiltersValues.tsx

```diff
@@ -91,6 +132,30 @@ const useFiltersValues = (
     search,
     setFilterValue,
   };
+
+  return (
+    <FiltersContext.Provider value={filterValues}>
+      {children}
+    </FiltersContext.Provider>
+  );
 };
 
-export { useFiltersValues };
+const withFiltersContext = ({
```

Comment:
Honestly, having worked with codebases that had multiple tens of lines wrapping stuff before exporting at the bottom of the file, I would really like to avoid it if possible ^^ it really lowers legibility, in my experience. Having only one component defined in a file, and exporting it wrapped in a `withFiltersContext` feels like the best compromise to me.

Reply:
I mean, I can live with it! It's basically the same thing and in case we encounter some readability issue in the future, we still can revisit it!